feature/experimental/tunnel/otg_tests/tunnel_interface_based_resize_test/README.md

```diff
@@ -0,0 +1,37 @@
+
+
+
+ ## TUN-1.6: Tunnel End Point Resize - Interface Based GRE Tunnel
+
+ ## Summary
+
+ *    Tunnel End Point Resize - Interface Based GRE Tunnel
+
+ ## Procedure
+
+ *   Configure DUT with 16 GRE encapsulation tunnels and configure another router with 12 GRE Decapsulation tunnel interfaces.
+ *   Configure 8 tunnel as IPv4 tunnel source and destination address , 8 as IPv6 tunnel source and destination address
+ *   Configure static router to point original destination to 16 tunnel interface to do overlay loadbalance
+ *   Keep topology tunnel destination will be reachable via two underlay interface on both routers
+ *   Send IPv4 flow and IPv6 flow and validate tunnel load balance and physical interface load balance
+ *   resize the tunnel fro 16 to 12 and verify the load balance and traffic drop by removing static route to point tunnel interface.
+ *   Again resize the tunnel fro 12 to 16 and verify the load balance and traffic drop
```

Comment:
This is what the Original testplan says

```
- Apply the config mentioned in Tunnel-1.1
- Send the traffic as mentioned in Tunnel-1.3 and Tunnel-1.4 with TP-1.1 and TP-1.2
- Modify the numbers of Tunnel interface being used 
   - Reduce number of Tunnel interfaces(e.g. From 16 to 12):
         - Incoming traffic on DUT-PORT1 should be load balanced to available Tunnel interfaces for encapsulation
         - Incoming traffic flow should be equally distributed for Encapsulation(ECMP)
- If the static routes are used to forward traffic to tunnel, please disable or delete the static route in this test to simulate the reduction in available paths
  - Increase number of Tunnel interfaces(e.g. From 16 to 20): 
  - Incoming traffic on DUT-PORT1 should start using additional Tunnel Interfaces  for encapsulation
  - Incoming traffic flow should be equally distributed for Encapsulation(ECMP)

- Increasing and decreasing the number of tunnel interfaces and related static route shouldn’t case traffic drops

Verify the next hop counters for packet being diverted or sent for encapsulation

Verify the tunnel interfaces counters to confirm the traffic encapsulation

Verify the tunnel interfaces traffic/flow for equal distribution for optimal load balancing

After decapsulation, traffic should be load balanced/hash to all available L3 ECMP or LAG or combination of both features

Verify the tunnel interfaces counters to confirm the traffic decapsulation

Validate system for:
Health-1.1
No feature related error or drop counters incrementing, discussion with vendors required to highlight additional fields to monitor based on implementation and architecture

Note: System level load sharing/hashing of GRE Encap packet should be covered under load balance/hashing function test case
```

Reply:
Hi sachendras,

I have corrected "README.md" file as per the Testplan 
Also I have corrected "CONTRIBUTING.md" error and "static Analysis" error 

Regards,
Senthil Kumar